structopt-derive/COPYING

```diff
@@ -1 +0,0 @@
-../COPYING
```

Comment:
Please symlink the LICENSE-* files in this directory as asked in #61

Reply:
I'm on Windows, and this appeared as just a text file for me (and GitHub apparently).

The easiest (and common) solution is just to duplicate the license files, so unless you come up with a better idea that works through Git, I'll just do that (in a separate commit).

Reply:
Copying the file is OK for me.

Reply:
It's done now! 👍

Reply:
Great, approved.